### ISSUE TITLE
Require Chromium backend

### ISSUE THREAD

**Issue by jiri-janousek** [tiliado/nuvola-app-amazon-cloud-player#34]
Hello @a-m-s, could you modify the metadata to require the Chromium backend and test it again? I've recently implemented URL filtering for it, so it might break something.

```
"#requirements": "https://github.com/tiliado/nuvola-app-amazon-cloud-player/issues/33",
"requirements": "Chromium[65] Codec[MP3] Feature[MSE] Feature[Flash]",
```

**Comment by a-m-s** [comment]
I now get a message that Flash is disabled. :-(

It works fine without Chromium.

**Comment by jiri-janousek** [comment]
Could you attach a debugging log?

**Comment by a-m-s** [comment]
So, my machine crashed badly. Corrupt root filesystem. Everything gone. Nightmare......

Despite running Arch Linux for years I couldn't get the graphics drivers to work again, and I need to get work done, so I've installed Ubuntu 17.10. I lost the Flatpak installs, but my user data and Nuvola sources are all safe on my home partition, so I just followed the wiki instructions again to reinstall the Flatpaks, and I'm up and running again.

.... and now the Amazon Player is **working perfectly on Chromium** with the features you quoted. Flash is present and working properly.

I'm not sure what changed? Maybe I had something installed from the master branch before?
